driver/ppm_events_public.h

```diff
@@ -1326,6 +1338,7 @@ struct ppm_evt_hdr {
 	uint64_t tid; /* the tid of the thread that generated this event */
 	uint32_t len; /* the event len, including the header */
 	uint16_t type; /* the event type */
+	uint32_t nparams; /* the number of parameters of the event */
```

Comment:
There seem to be a few places where `nparams` is left uninitialized, it would be good to either fix those or acknowledge the lack of by maybe leaving a comment (or it's just ok if you just ack here that you checked them). The places I can see are:

```
capture_job_handler::push_notification()
```
```
capture_job_handler::push_infra_event()
```
```
schedule_more_evts
```

Reply:
I must have missed them when I updated the codepaths that creates an event block, will fix